website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,244 @@
+
+1. Ensure your dbt project is in a Git repository
+
+2. Once you’ve set up a dbt Cloud account, you can [connect and configure Git](/docs/cloud/git/git-configuration-in-dbt-cloud) in dbt Cloud to connect your Git repository:
+   - Connect with one of the [native integrations](/docs/cloud/git/git-configuration-in-dbt-cloud) in dbt Cloud (such as GitHub, GitLab, and Azure DevOps).
+   - [Import a git repository](/docs/cloud/git/import-a-project-by-git-url) from any valid git URL that points to a dbt project.
+
+### Advanced configuration
+3. Log into dbt Cloud using [OAuth connections](/docs/cloud/git/connect-github) to integrate with your source code platform. It automatically links to the repository using one of the native integrations set at the account level. <Lifecycle status='enterprise' />
+  
+  Set up groups for dbt project access with those configured for repository access to streamline permissions.
+
+## Developer setup
+
+This section highlights the development configurations you’ll need for your dbt Cloud project. The following categories are covered in this section:
+
+- [dbt Cloud environments](/guides/core-to-cloud-1?step=7#dbt-cloud-environments)
+- [Initial setup steps](/guides/core-to-cloud-1?step=7#initial-set-up-steps)
+- [Advanced configuration](/guides/core-to-cloud-1?step=7#initial-set-up-steps)
+- [dbt Cloud commands](/guides/core-to-cloud-1?step=7#dbt-cloud-commands)
+
+### dbt Cloud environments
+
+The concept of a `target` in dbt Core is the same as a [dbt Cloud environment](/docs/environments-in-dbt).
+
+The main difference between `target` in dbt Core and a dbt Cloud environment is that you can make these configurations through the dbt Cloud UI, as opposed to within your **`profiles.yml`** file.
+
+This difference streamlines the process of switching between development, staging, and production contexts, removing the need to manually edit a **`profiles.yml`** file. dbt Cloud environments also integrate with additional features such as job scheduling, version control, and more, making it easier to manage the full lifecycle of your dbt projects within a single platform You can [set up](/reference/dbt-jinja-functions/target) or [customize](/docs/build/custom-target-names) target names in dbt Cloud.
+
+### Initial setup steps
+1. **Set up development environment** &mdash; Set up your [development](/docs/dbt-cloud-environments#create-a-development-environment) environment and [development credentials](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud#access-the-cloud-ide). You’ll need this to access your dbt project and start developing.
+
+2. **dbt Core version** &mdash; In your dbt Cloud environment and credentials, use the same dbt Core version you use locally. You can run `dbt --version` in the command line to find out which version of dbt Core you’re using.
```

Comment:
When will new accounts all be versionless? We may want to make a note to update this step after that date.
```suggestion
2. **dbt Core version** &mdash; In your dbt Cloud environment and credentials, use the same dbt Core version you use locally. You can run `dbt --version` in the command line to find out which version of dbt Core you’re using.
```

Reply:
yep for sure! this is a common issue users find themselves in when they're migrating per the smes so added this in as this is an important detail to note for them.